libs/domain/cart/entry/src/entry.component.ts

```diff
@@ -6,23 +6,24 @@ import {
   ProductMediaContainerSize,
   ProductMixin,
 } from '@spryker-oryx/product';
+import { RouteType } from '@spryker-oryx/router';
```

Comment:
Do we need it?

Reply:
yeah, we use it in:
```ts
protected $productLink = computed(() => {
    return this.semanticLinkService.get({
      type: RouteType.Product,
      id: this.$product()?.sku,
    });
  });
```

not sure why the PR changes do not show it.